contributing.md

```diff
@@ -16,37 +16,51 @@ Documentation for Tenzir projects can be found at
 
 ## Git and GitHub Workflow
 
-At Tenzir, our git workflow encompasses the following key aspects.
+The following sketch visualizes our branching model:
+
+![Branching Model](https://user-images.githubusercontent.com/53797/156560785-c7279447-63eb-4428-9a11-9c90cc03acc8.png)
+
+Our git workflow looks as follows:
 
 - The `master` branch reflects the latest state of development, and should
   always compile.
 
+- In case we need to release a hotfix, we use dedicated patch release branches.
+
+- The stable `branch` always points to the latest release that is not a release
+  candidate. CI resets it automatically. It exists so support a streamlined
```

Comment:
```suggestion
  candidate. It exists so support a streamlined
```

We do this manually. We definitely don't want to allow CI to force-push, like ever.